src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-list/nfs-list.component.html

```diff
@@ -29,6 +29,20 @@
                 i18n>Object Gateway</ng-container>
 </ng-template>
 
+<ng-template #protocol
+             let-row="data.row">
```

Comment:
this will give you `protocols` since that's what you are passing. so you can just use `protocols` instead of `row.protocols` on the `*ngFor`
```suggestion
             let-protocols="data.value">
```

Reply:
updated